_ml-commons-plugin/using-ml-models.md

```diff
@@ -19,6 +19,9 @@ To integrate machine learning (ML) models into your OpenSearch cluster, you can
 
 - **Custom models** such as PyTorch deep learning models: To learn more, see [Custom models]({{site.url}}{{site.baseurl}}/ml-commons-plugin/custom-local-models/).
 
+OpenSearch does not support running local models on the CentOS 7 operating system. Moreover, not all local models can run on all hardware and operating systems.
```

Comment:
I don't think we should generalize this to OpenSearch rather to ml-commons plugin. Is this OpenSearch limitation or ml-commons plugin limitation?

Reply:
Good point, this should be ml-commons plugin

Reply:
I'm open to changing to ML Commons, however, in general, we do not mention plugins in the documentation, and since there is no other way to run models in OpenSearch, maybe keeping it general is OK. @bbarani @ylwu-amzn Let me know your thoughts.

Reply:
I see! Its just that we explicitly mention that we support CentOS 7 for OpenSearch [here](https://opensearch.org/docs/latest/install-and-configure/install-opensearch/index/#operating-system-compatibility) and this statement contradicts it.

Reply:
OK, let me make it more generic by using passive voice instead of mentioning the plugin.